openapi/table_openapi_path_request_body.go

```diff
@@ -81,16 +81,24 @@ func listOpenAPIPathRequestBodies(ctx context.Context, d *plugin.QueryData, h *p
 
 			for header, content := range operation.RequestBody.Value.Content {
 				var schema interface{}
-				if content.Schema.Ref != "" {
-					schema = content.Schema.Ref
-				} else {
-					schema = content.Schema
+				var schemaType string
+
+				// Check if content.Schema and content.Schema.Value is not nil
```

Comment:
@bigdatasourav In 3.0 documents, would these scenarios ever happen, or is it only likely in 2.0 documents?

Reply:
@cbruno10, we got these scenarios in the 3.0 documents during our testing.